warehouse/core/src/main/java/datawave/util/StringUtils.java

```diff
@@ -1,10 +1,6 @@
 package datawave.util;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Iterator;
-import java.util.List;
-import java.util.NoSuchElementException;
+import java.util.*;
```

Comment:
Don't import with wildcards, please.

Reply:
Funny. I did not make that change deliberately. Most probably Intelij did it. I will fix it.